### PR TITLE
mdcat: install completions for mdless.

### PR DESCRIPTION
**Pull request by mhmdanas**
#### Testing the changes
- I tested the changes in this PR: **YES**

@cinerea0

### CODE REVIEW

File: srcpkgs/mdcat/template [hunk 1]
```diff
@@ -22,14 +22,18 @@ case "$XBPS_TARGET_MACHINE" in
...
+		vcompletion ${zsh_completion} zsh ${command}
+		fish_completion=$(find ${wrksrc}/target -name ${command}.fish -print -quit)
+		vcompletion ${fish_completion} fish ${command}
+	done
 
 	man_page=$(find ${wrksrc}/target -name mdcat.1 -print -quit)
 	vman ${man_page}
+
 	ln -sf /usr/bin/mdcat ${DESTDIR}/usr/bin/mdless
+	ln -sf mdcat.1 ${DESTDIR}/usr/share/man/man1/mdless.1
```
Comment (classabbyamp) [inline srcpkgs/mdcat/template:38]: i don't see any other distro doing this, and the help output for mdless says `See 'man 1 mdcat' for more information.`

Comment (mhmdanas) [reply srcpkgs/mdcat/template:38]: Still, here's the synopsis for the manpage:
```
       mdcat [OPTIONS] [FILE]...

       mdless [OPTIONS] [FILE]...
```

So I think it makes sense.

Comment (sgn) [reply srcpkgs/mdcat/template:38]: Nope, I don't think it make sense, its `.TH` is `MDCAT`, its `.SH "NAME"`  is `mdcat`, upstream should change like this:
```diff
30c30
< mdcat \- render CommonMark Markdown to text terminals
---
> mdcat, mdless \- render CommonMark Markdown to text terminals
```

Comment (mhmdanas) [reply srcpkgs/mdcat/template]: I understand, I removed the manpage symlink for now.

**Pull request merged**